docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -160,10 +162,39 @@ The following configuration properties are available:
        This flag allows ignoring the value provided in the metadata document.
        Default is ``true``.
 
+.. _trino-oauth2-refresh-tokens:
+
 Refresh tokens
 ^^^^^^^^^^^^^^
 
-In order to start using refresh tokens flows, the following property must be
+*Refresh tokens* allow you to securely control the length of user sessions
+within applications. The refresh token has a longer lifespan (ttl) and is used
+to refresh the *access token* that has a shorter lifespan. With refresh tokens
+behind the scenes, users can stay logged in for a longer session without
+repeating the authentication process.
+
+In a refresh token flow, there are three tokens with different expiration times:
+
+* access token
+* refresh token
+* Trino-encrypted token that contains both an access token and a refresh token.
```

Comment:
of the access and refresh token*s* (plural)

Reply:
@bendemott incorporated your comment with some edits. Did not want to repeat mention of encrypted within the same sentence, I try to avoid somewhat blind subject references like "this" or "it", and am trying not to use "via" (Latin). @m57lyra plural makes sense here; inc. that as well.